mlir/lib/Dialect/MQTDyn/Transforms/ConstantFolding.cpp

```diff
@@ -36,7 +35,7 @@ struct ConstantFolding final : impl::ConstantFoldingBase<ConstantFolding> {
     populateConstantFoldExtractQubitPatterns(patterns);
 
     // Apply patterns in an iterative and greedy manner.
-    if (mlir::failed(mlir::applyPatternsGreedily(op, std::move(patterns)))) {
+    if (mlir::failed(APPLY_PATTERNS_GREEDILY(op, std::move(patterns)))) {
```

Comment:
The `#include <mlir/Support/LLVM.h>` include was erroneously removed from this file.